src/components/MessagesList/MessagesGroup/MessagesSystemGroup.vue

```diff
@@ -0,0 +1,456 @@
+						break
+					default :
+						combinedMessage.message = t('spreed', '{actor} added {user0}, {user1} and {hiddenUsers} more participants', { hiddenUsers: usersCounter - 2 })
+						break
+					}
+				}
+			}
+
+			// Handle cases when users reconnected to the call
+			if (type === 'call_reconnected') {
+				if (this.checkIfSelfIsActor(combinedMessage)) {
+					combinedMessage.message = t('spreed', 'You reconnected to the call')
+				} else {
+					combinedMessage.message = t('spreed', '{actor} reconnected to the call')
+				}
+			}
+
+			// Handle cases when users joined the call
+			if (type === 'call_joined') {
+				messages.forEach((message, index) => {
+					if (this.checkIfSelfIsActor(message)) {
+						selfIsUser = true
+					} else {
+						combinedMessage.messageParameters[`user${referenceIndex}`] = message.messageParameters.actor
+						referenceIndex++
+					}
+					usersCounter++
+				})
+
+				if (selfIsUser) {
+					switch (usersCounter) {
+					case 2 :
+						combinedMessage.message = t('spreed', 'You and {user0} joined the call')
+						break
+					case 3 :
+						combinedMessage.message = t('spreed', 'You, {user0} and 1 more participant joined the call')
+						break
+					default :
+						combinedMessage.message = t('spreed', 'You, {user0} and {hiddenUsers} more participants joined the call', { hiddenUsers: usersCounter - 2 })
+						break
+					}
+				} else {
+					switch (usersCounter) {
+					case 2 :
+						combinedMessage.message = t('spreed', '{user0} and {user1} joined the call')
+						break
+					case 3 :
+						combinedMessage.message = t('spreed', '{user0}, {user1} and 1 more participant joined the call')
+						break
+					default :
+						combinedMessage.message = t('spreed', '{user0}, {user1} and {hiddenUsers} more participants joined the call', { hiddenUsers: usersCounter - 2 })
+						break
+					}
+				}
+			}
+
+			// Handle cases when users left the call
+			if (type === 'call_left') {
+				messages.forEach((message, index) => {
+					if (this.checkIfSelfIsActor(message)) {
+						selfIsUser = true
+					} else {
+						combinedMessage.messageParameters[`user${referenceIndex}`] = message.messageParameters.actor
+						referenceIndex++
+					}
+					usersCounter++
+				})
+
+				if (selfIsUser) {
+					switch (usersCounter) {
+					case 2 :
+						combinedMessage.message = t('spreed', 'You and {user0} left the call')
+						break
+					case 3 :
+						combinedMessage.message = t('spreed', 'You, {user0} and 1 more participant left the call')
+						break
+					default :
+						combinedMessage.message = t('spreed', 'You, {user0} and {hiddenUsers} more participants left the call', { hiddenUsers: usersCounter - 2 })
+						break
+					}
+				} else {
+					switch (usersCounter) {
+					case 2 :
+						combinedMessage.message = t('spreed', '{user0} and {user1} left the call')
+						break
+					case 3 :
+						combinedMessage.message = t('spreed', '{user0}, {user1} and 1 more participant left the call')
```

Comment:
that's not how plurals work :-X

Reply:
:see_no_evil: Sure, will overwrite with `n('spreed')` later